RELEASE-GUIDE.md

```diff
@@ -0,0 +1,63 @@
+# How to release
+
+This documentation explains the process to release the design system.
+
+## Create a release candidate
+
+First of all, before releasing a new version, you need to create a release candidate. This is a version that is not yet released but that can be tested by the users.
+
+To create a release candidate, you need to create a new branch from the `main` branch. The branch name should be `releases-DESIGN_SYSTEM/X.Y.Z` where `DESIGN_SYSTEM` is the design system you want to release and `X.Y.Z` is the version number you want.
+
```

Comment:
Donnner le nom exact des branches (confusion possible entre agent/slash ou look-and-feel/lookandfeel/apollo)